src/App.tsx

```diff
@@ -1,10 +1,65 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+// import { TodoList } from './TodoList';
 
-const USER_ID = 0;
+const USER_ID = 10995;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [err, setErr] = useState<string | null>(null);
+  const [filterMethod, setFilterMethod] = useState<string>('All');
+  const [isHidden, setIsHidden] = useState(false);
+
+  let visibleTodos = [...todos];
+
+  switch (filterMethod) {
+    case 'Completed':
+      visibleTodos = [...visibleTodos].filter(todo => todo.completed === true);
```

Comment:
```suggestion
      visibleTodos = visibleTodos.filter(todo => todo.completed === true);
```
Filter is a non-mutating method. There is no need to create one more copy